daml-script/test/daml/MultiTest.daml

```diff
@@ -47,10 +47,12 @@ listKnownPartiesTest : Script ([(Optional Text, Bool)], [(Optional Text, Bool)])
 listKnownPartiesTest = do
   parties1 <- listKnownPartiesOn (ParticipantName "one")
   parties2 <- listKnownPartiesOn (ParticipantName "two")
-  p1 <- allocatePartyOn "p1" (ParticipantName "one")
   p2 <- allocatePartyOn "p2" (ParticipantName "two")
+  p1 <- allocatePartyOn "p1" (ParticipantName "one")
```

Comment:
was this the fix or was it the portfile change?

Reply:
See https://daholdings.slack.com/archives/CG120JNQ0/p1678353306836319

Reply:
both.

Reply:
(both)

Reply:
Did not fully check for the portfile change though, but it fixed flakyness in other cases I had.
In particular it happened the port is listening but the domain is not ready behind, leading to submission failure.